src/core/index.ts

```diff
@@ -135,11 +137,21 @@ export class CrowdinValidationError extends CrowdinError {
 /**
  * @internal
  */
-function handleError<T>(error: any = {}): T {
-    if (Array.isArray(error.errors)) {
+function handleHttpClientError(error: unknown): never {
```

Comment:
I guess now we can define type here `error: AxiosError | FetchClientJsonPayloadError | Error`. What do you think?

Reply:
we sure could. however, the way this function is used right now is .catch cb's pass in concrete `err: unknown` values. if we update this signature, we will have to convert or every `err: unknown` in each associated catch. i DO prefer your suggestion, but it may bloat the changeset a bit.

let me give it a shot.

Reply:
ah, i was wrong, `.catch(e => ...)` `e` is `any`, not `unknown` by default in typescript. very obnoxious :|! in `try { } catch (err) { }`, `err` is `unknown`. a divergence. ts isn't perfect :)

nonetheless, this is an improved impl that achieves the request: 338a531